lab_1/main.py

```diff
@@ -2,40 +2,62 @@
 Lab 1
 A concordance extraction
 """
+
 text = open('data.txt', encoding = 'utf-8').read()
 
-# ФУНКЦИЯ РАБОТАЕТ, НО ПОСМОТРЕТЬ, ПОЧЕМУ  ТЕСТЫ НЕ ВИДЯТ ФАЙЛ "TOKENS.TXT" + '`the'
-def tokenize(text):
+
+def tokenize(text: str) -> list:
+    """
+    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
+    :param text: the initial text
+    :return: a list of lowercased tokens without punctuation
+    e.g. text = 'The weather is sunny, the man is happy.'
+    --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
+    """
     import re
     if type(text) == str:
         new_text = ''
         for elem in text:
             if elem.isalpha() or re.fullmatch('[\s]', elem):
                 new_text += elem
-        new_text = new_text.lower()
-        new_text = new_text.split()
+        new_text = new_text.lower().split()
         return new_text
     else:
         return []
 
 list_of_tokens = tokenize(text)
 
-
 stop_words = open('stop_words.txt').read()
```

Comment:
you need to write it in start.py
main.py is only for function implementation
no prints, no reading from files